tools/junitreport/pkg/parser/gotest/data_parser.go

```diff
@@ -10,13 +10,13 @@ func newTestDataParser() testDataParser {
 	return testDataParser{
 		// testStartPattern matches the line in verbose `go test` output that marks the declaration of a test.
 		// The first submatch of this regex is the name of the test
-		testStartPattern: regexp.MustCompile(`=== RUN\s+(.+)$`),
+		testStartPattern: regexp.MustCompile(`=== RUN\s+([^/]+)$`),
```

Comment:
If there is always a function name, then I don't want to treat start of a subtest as a start of test.